Analysis/include/SyncTuple.h

```diff
@@ -5,6 +5,7 @@ This file is part of https://github.com/hh-italian-group/h-tautau. */
 
 #include "AnalysisTools/Core/include/SmartTree.h"
 #include "AnalysisTools/Core/include/AnalysisMath.h"
+#include "EventInfo.h"
```

Comment:
SyncTuple.h is an old definition of SyncTuple, which is no longer supported. Please, remove it.